README.md

```diff
@@ -10,14 +10,123 @@
+
+* vendor – in earlier versions of Rails the public folder used to be where developers could place assets, such as CSS styles and JavaScript code, but in Rails 4+ the Rails asset pipeline responsibilities were moved to the app/assets directory. In Rails 4+ the vendor directory can be utilized to manage outside JavaScript frameworks.
+
+* Gemfile – the Gemfile contains all of the gems that are included in the application; this is where you will place outside libraries that are utilized in the application. After any change to the Gemfile you will need to run: bundle. This will call in all of the code dependencies into the application. The Gem process can seem like a mystery to new developers, but it is important to realize that the Gems that are brought into an application are simply Ruby files that help extend the functionality of the app.
+
+* Gemfile.lock – this file should not be edited; it displays all of the dependencies that each of the Gems contain along with their associated versions. There have been times where I discovered application bugs due to varying Gem’s dependencies.
+
+* README.rdoc – the readme file is an important place to document the details of the application. If the application is an open-source project, this is where you can place instructions to other developers, such as how to get the app up and running locally.
+
+## Creating the database
+
+Before we can startup the rails server, first you can create the database by running:
+
+```rake db:create```
+
+## Starting up the Rails Server
+
+To startup the Rails server, make sure that you are in the root of the application in the terminal and run:
+
+```rails s```
+
+This will startup the rails server and you will see output such as the following:
+
+```
+=> Booting WEBrick
+=> Rails 4.2.3 application starting in development on http://localhost:3000
+=> Run `rails server -h` for more startup options
+=> Ctrl-C to shutdown server
+[2015-11-14 22:16:54] INFO  WEBrick 1.3.1
+[2015-11-14 22:16:54] INFO  ruby 2.1.2 (2014-05-08) [x86_64-darwin13.0]
+[2015-11-14 22:16:54] INFO  WEBrick::HTTPServer#start: pid=3080 port=3000
+```
+
+Now that the server is running properly, you can go and verify that it's working properly in the browser by going to: ```http://localhost:3000/```
+
+Here you will see the 'Welcome aboard' page that ships with Rails and it shows that we're ready to start building the application!
+
+## Using the Rails Console
+
+The Rails console is an important tool in the arsenal for any Rails developer, it gives you a direct connection to your application's ecosystem and lets you perform tasks such as:
+
+* Run database queries
+* Run application code
+* Perform full CRUD tasks with the database
+* Allows you to switch between making permanent database changes and running in a sandbox mode to test scripts out.
+
+To startup the rails console, run the following command in the terminal:
+
+```rails c```
```

Comment:
I think we need to move rails c into a new reading a bit down the road once they know some stuff. It's not obvious why rails c is better than irb right now. We'd need to create a simple model and then show them that if you do irb you can't access it, if you do rails c it loads your environment in and then you can access all of the models!